client/src/pages/home/views.tsx

```diff
@@ -0,0 +1,205 @@
+        )?.teams ?? [],
+        p2.assignmentContributors.find(
+          (contributor) => contributor.userId === user._id,
+        )?.teams ?? [],
+      ),
+    extractor: function TeamsCell(pitch) {
+      return <PitchTeams pitch={pitch} user={user} assignmentContributors />;
+    },
+  },
+  {
+    title: 'Deadline',
+    width: '1',
+    sorter: (p1, p2) =>
+      new Date(p1.deadline).getTime() - new Date(p2.deadline).getTime(),
+    extractor: (pitch) => new Date(pitch.deadline).toLocaleDateString(),
+  },
+];
+
+const getSubmittedPitchesView = (): ColumnType<IPitch>[] => [
+  titleColumn,
+  descriptionColumn,
+  {
+    ...associatedTopicsColumn,
+    width: '2',
+  },
+  {
+    title: 'Status',
+    width: '1',
+    sorter: (p1, p2) => p1.status.localeCompare(p2.status),
+    extractor: function StatusCell({ status }) {
+      return <FieldTag content={status} />;
+    },
+  },
+  {
+    title: 'Date Submitted',
+    width: '1',
+    sorter: (p1, p2) =>
+      new Date(p1.createdAt).getTime() - new Date(p2.createdAt).getTime(),
+    extractor: function DateCell(pitch) {
+      return new Date(pitch.createdAt).toLocaleDateString();
+    },
+  },
+];
+
+const getSubmittedClaimsView = (user: IUser): ColumnType<IPitch>[] => [
+  {
+    ...titleColumn,
+    width: '4',
+  },
+  {
+    ...associatedTopicsColumn,
+    width: '2',
+  },
+  {
+    title: 'Team(s) Requested to Claim',
+    width: '2',
+    sorter: (p1, p2) =>
+      stringArraySorter(
+        p1.pendingContributors.find((request) => request.userId === user._id)
+          ?.teams ?? [],
+        p2.pendingContributors.find((request) => request.userId === user._id)
+          ?.teams ?? [],
+      ),
+    extractor: function TeamsCell(pitch) {
+      return <PitchTeams pitch={pitch} user={user} />;
+    },
+  },
+  {
+    title: 'Status',
+    width: '1',
+    sorter: (p1, p2) => p1.status.localeCompare(p2.status),
+    extractor: function StatusCell() {
+      return <FieldTag content={'Pending'} />;
+    },
+  },
+  {
+    title: 'Date Submitted',
+    width: '1',
+    sorter: (p1, p2) =>
+      new Date(
+        p1.pendingContributors.find(
+          (contributor) => contributor.userId === user._id,
+        )?.dateSubmitted ?? Date.now(),
+      ).getTime() -
+      new Date(
+        p2.pendingContributors.find(
+          (contributor) => contributor.userId === user._id,
+        )?.dateSubmitted ?? Date.now(),
+      ).getTime(),
+    extractor: function DateCell(pitch) {
+      return new Date(
+        pitch.pendingContributors.find(
+          (contributor) => contributor.userId === user._id,
+        )?.dateSubmitted ?? Date.now(),
+      ).toLocaleDateString();
+    },
+  },
+];
+
+const getSubmittedPublicationsView = (): ColumnType<IIssueAggregate>[] => [
+  {
+    title: 'Title',
+    width: '3',
+    sorter: (i1, i2) => i1.name.localeCompare(i2.name),
+    extractor: 'name',
+  },
+  {
+    title: 'Associated Topics',
+    width: '1',
+    sorter: (i1, i2) =>
```

Comment:
We don't need to sort by Interests, Teams, or Description. This is good stuff, but make sure you are referencing how the tables in other parts already work just to make sure you don't give yourself extra work because I don't want you working on things that you can't keep.